wolfssl/test.h

```diff
@@ -355,11 +355,29 @@ void join_thread(THREAD_TYPE);
 #endif
 static const word16      wolfSSLPort = 11111;
 
-static INLINE void err_sys(const char* msg)
+
+#if defined(__GNUC__)
+    #define WC_NORETURN __attribute__((noreturn))
+#else
+    #define WC_NORETURN
+#endif
+
```

Comment:
I considered that but... I thought we shouldn't consider no return hacks in the library proper.  Library functions should return, we should leave exit() and abort() to the user.  For testing though, where we ARE the user, it makes more sense for this hack.